### PR TITLE
Folder restructure to avoid Xcode recents issue

### PR DESCRIPTION
**Pull request by osanoj**
# Why?
Because when having the xcworkspace in the same folder as the Package.swift file Xcode will choose the package for the list of recently opened files despite you having used the workspace before. 

# What?
- Moved workspace to `Workspace/FinnUIDemo.xcworkspace`. 
- Added instructions to use this for development into README. 
- Added an empty (almost) Package.swift to the Demo-folder, that way that folder will be hidden from the main package.

# Version Change
Nope, only changes to dev setup.

### CODE REVIEW

**Review by hakloev — APPROVED** [review]

File: Demo/Package.swift [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// swift-tools-version:5.3
```
Comment (bstien) [inline Demo/Package.swift:1]: Is this file needed?

**Review by bstien — APPROVED** [review]
LGTM

File: Demo/Package.swift (hunk 1 above)
Comment (osanoj) [reply Demo/Package.swift:1]: It is a sort of hack to exclude that folder from the normal package
"Added an empty (almost) Package.swift to the Demo-folder, that way that folder will be hidden from the main package"

So instead of:
Screen Shot 2021-02-05 at 14 15 20

You get:
Screen Shot 2021-02-05 at 14 13 31

Comment (osanoj) [reply Demo/Package.swift:1]: Found that workaround here: https://forums.swift.org/t/hiding-ignoring-directories-from-xcode-when-opening-swift-packages/35431/6

Comment (bstien) [reply Demo/Package.swift:1]: Weird hack, but it definitely makes the overview much easier!
